EnglishFilter/sections/adservers.txt

```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||oplwseln.com^$third-party
```

Comment:
this adserver is already blocked by the base filter but I found 2 more:
```
||betjeyxf.com^$third-party
||bbocajxn.com^$third-party
```

and i think we should add movie8k.* to the rule under this comment:

! For URL shorteners or simple sites with frequently changed ad servers.
! Incorrectly blocked domains can be excluded using regex(synchronize this regexp in all filters)

Reply:
Yes, they changed their code. Let me update all.

Reply:
Fixed.